lib/workers/repository/process/types.ts

```diff
@@ -0,0 +1,19 @@
+import type { Osv } from '@renovatebot/osv-offline';
+import type { RenovateConfig } from '../../../config/types';
+import type { PackageFile } from '../../../modules/manager/types';
+import type { VersioningApi } from '../../../modules/versioning';
+
+export interface Vulnerability {
+  packageFileConfig: RenovateConfig & PackageFile;
+  packageName: string;
+  depVersion: string;
+  fixedVersion: string | null;
+  datasource: string;
+  vulnerability: Osv.Vulnerability;
+  affected: Osv.Affected;
+}
+
+export interface VulnerabilityGroup {
```

Comment:
`VulnerabilityGroup` is actually representing a list of vulnerabilities + the versioning api of a particular package file, correct?

I find the term ambiguous and unintuitive, as it isn't clear what is represented by this group. As a security engineer, I'd assume e.g. a group of _memory corruption vulnerabilities_ or think they are somehow related (same severity, same vulnerable component, etc). 

Suggestion: Let's rename this to `PackageFileVulnerabilities` instead?

Reply:
It's a grouping in the sense we need to sort this group of Vulnerabilities.   

I see your point. 
Regarding `PackageFileVulnerabilities`. `VulnerabilityGroup` is not based on `packageFile` but rather a dependency.  

I will rename it to `DependencyVulnerabilties`.

Reply:
Thanks, it really wasn't obvious to me that there would a group in this case is a list of vulns in a specific dependency.